app/views/records/edit_fields/_preservation_policy.html.erb

```diff
@@ -0,0 +1,8 @@
+  <%= f.input key,
+              as: :select,
+              collection: ControlledVocabulary.for(key).all,
+              label_method: :label,
+              value_method: :value,
+              include_blank: true,
+              input_html: { class: 'form-control rights-statement' },
+              required: f.object.required?(key) %>
```

Comment:
We have a few partials that are basically identical to this (e.g., `_holding_location.html.erb` or `_owners.html.erb`). Do you think it's worth making a helper?

Reply:
Probably, maybe as another issue though?

Reply:
👍 I created a `refactor` ticket for that: https://github.com/pulibrary/figgy/issues/2834